test/propagation_test.cpp

```diff
@@ -900,7 +900,9 @@ TEST_CASE("propagated Datadog tags (x-datadog-tags)") {
       REQUIRE(result);
 
       const auto injected_json = nlohmann::json::parse(injected.str());
-      REQUIRE(injected_json.find("tags") == injected_json.end());
+      const auto tags = injected_json.find("tags");
+      // See <https://github.com/DataDog/dd-opentracing-cpp/issues/241>.
+      REQUIRE((tags == injected_json.end() || *tags == ""));
```

Comment:
Maybe a comment about why double-brackets were needed for future-me or future-you.